project/Version.scala

```diff
@@ -33,7 +33,7 @@ object Version {
   val lettuce                = "6.2.4.RELEASE"
   val java8Compat            = "1.0.2"
   val scalaMock              = "4.3.0"
-  val guava                  = "33.0.0-jre"
+  val guava                  = "32.0.0-android"
```

Comment:
there aren't any concerns switching between the JRE and Android version of this library?

Reply:
Hey @jaredb96 , checking https://github.com/google/guava and it does seem like the "-jre" version is intended for JRE-based builds. Looks like guava is up to v33.3.1. What does grype say if you put in 33.3.1-jre?

Reply:
OK, looks like scala itself was bringing in the android dep! That's weird... This removes the guava problem from the grype output.

```
-      libraryDependencies += zeroAllocationHashing,
+      libraryDependencies ++= List("com.google.guava" % "guava" % "33.3.1-jre", zeroAllocationHashing),
```